homeassistant/components/palazzetti/config_flow.py

```diff
@@ -0,0 +1,57 @@
+"""Config flow for Palazzetti."""
+
+from typing import Any
+
+from pypalazzetti.client import PalazzettiClient
+from pypalazzetti.exceptions import CommunicationError
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigFlow, ConfigFlowResult
+from homeassistant.const import CONF_HOST, CONF_MAC, CONF_NAME
+from homeassistant.helpers import device_registry as dr
+
+from .const import DOMAIN, LOGGER
+
+
+class PalazzettiConfigFlow(ConfigFlow, domain=DOMAIN):
+    """Palazzetti config flow."""
+
+    VERSION = 1
+
+    async def async_step_user(
+        self, user_input: dict[str, Any] | None = None
+    ) -> ConfigFlowResult:
+        """User configuration step."""
+        if user_input and CONF_HOST in user_input:
```

Comment:
```suggestion
        if user_input is not None:
```
host is required, so this is all we need to check for